generate-api/src/main.rs

```diff
@@ -66,3 +68,32 @@ fn main() -> Result<()> {
 
     Ok(())
 }
+
+fn validate(objects: &mut Vec<Object>) {
```

Comment:
I feel like `validate()` is not a great name for something that takes a mutable reference?